pages/Contributing and Debugging/Issue-Guidelines.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Issue Guidelines
+---
+
+Inspired by Ghostty's approach to Github issues, we do not allow regular users to open issues anymore.
+
+This page is intended for people who want to open an **issue** on the **Hyprland** repo. Other hyprwm repositories
+still use issues normally.
+
+## Why?
+
+We are volunteers, doing this in our free time. Out of respect, please read this document
+_fully_ before posting an issue _or_ discussion. If you can spend a few minutes reading this,
+it saves us, the developers, much more time overall, so we can deliver fixes and features faster.
+
+Thank you!
+
+## Issue Guidelines
+
+Whatever your issue may be, _do not open an issue_. Instead, open a _discussion_ on the Hyprland repository.
+
+### Before you start a discussion
+
+_Search the discussions and issues thoroughly_. It's likely that there's already one open for your bug / feature that you can
+just "like" instead of wasting both your time writing a new one and our time closing it and redirecting you.
+
+### The lifecycle of a discussion
+
+A discussion is opened by a user.
+
+For bugs:
+- If a discussion is solved, user error, or not a hyprland issue, it gets closed.
+- If a discussion is a Hyprland problem, but cannot be tied to Hyprland in a clear way, or is not reproducible, it stays a discussion until that changes.
+- If a discussion is reproducible, a member of Hyprland promotes the discussion to an issue by opening an issue with the key information and links the original discussion.
+
+For feature requests:
+- If a discussion describes a feature request that is already possible, invalid, or not applicable, it gets closed.
+- If a discussion describes a feature that is _not_ already possible, it stays a discussion.
```

Comment:
Nit: These should probably have consistent wording, rather than saying "feature request" for the first line and just "feature" for the other two